src/pages/PassportRedemption/GiveawayPopover.tsx

```diff
@@ -0,0 +1,237 @@
+import React , { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import GreenCheck from './GreenCheck'
+import InstagramEnabled from './InstagramEnabled'
+import InstagramDisabled from './InstagramDisabled';
+
+import { getAllParticipatingSellers, getGiveawayTicketsForContact } from '../../utilities/api/interactionManager'
```

Comment:
nit: I prefer absolute over relative imports for easier greppability

Reply:
@justintmckibben I think we're using relative imports almost everywhere within the project 😬